scripts/export-txt/index.js

```diff
@@ -16,15 +16,26 @@ import { prepareDistFolder, BASE_PATH, getSpecs } from '../helpers.js';
   for (const [id, spec] of getSpecs('patterns')) {
     const category = spec.field('category').requiredStringValue();
     const filters = spec.field('filters').optionalStringValue();
+    const domains = spec.field('filters').optionalStringValue();
```

Comment:
This looks wrong. I assume, `domains` is meant instead of `filters`?

Reply:
good catch